src/deluge/gui/views/view.cpp

```diff
@@ -1385,6 +1399,17 @@ void View::drawOutputNameFromDetails(InstrumentType instrumentType, int32_t chan
 		                                              deluge::hid::display::OLED::oledMainImage[0],
 		                                              OLED_MAIN_WIDTH_PIXELS, kTextSpacingX, kTextSpacingY);
 	}
```

Comment:
when the automation instrument clip view is loaded or you change instrument types within the view the screen is re-rendered. Added this code to render the new 7SEG UI for the Automation Instrument Clip View.